src/Extensions/TimedScopes/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging.TimedScopes;
+
+namespace Microsoft.Omex.Extensions.TimedScopes
+{
+	/// <summary> Extension methods for the <see cref="IServiceCollection"/> class</summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>Add IServiceContext to ServiceCollection</summary>
```

Comment:
>summary [](start = 7, length = 7)

Add `<param>` and `<returns>` comments

Reply:
I don't think it makes sense here method is an extension that returns the same type